src/lib/field/Configurator/Select.vue

```diff
@@ -21,6 +29,11 @@ export default {
   data () {
     return {
       newOption: '',
+      selectOptions: [
+        { text: this.$t('field.kind.select.optionType.default'), value: 'default' },
```

Comment:
maybe https://panter.github.io/vue-i18next/guide/i18n-options.html#keyprefix to shorten this long i18n key